### PR TITLE
 📝 Improve gettingstarted readme

### PR DESCRIPTION
**Pull request by wvangeit**
## What do these changes do?

Update readme by removing Mac support and some getting started improvements

## Related issue/s

NA

## How to test

NA

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
Cool! Thx @wvangeit ! 
I added a suggestion and took the liberty to
- assign the issue
- add tags
- set milestone 
This is the minimal setup we like to have in every PR.
thx

**Review by elisabettai — APPROVED** [review]

**Review by bisgaard-itis — APPROVED** [review]
Looks good to me. Thanks for updating this.

File: README.md [hunk 1]
```diff
@@ -86,17 +83,19 @@ To develop, in addition:
 - swagger-cli (make sure to have a recent version of nodejs)
 - [vscode] (highly recommended)
 
-This project works and is developed under **linux (Ubuntu recommended)**.
+To verify current base OS, Docker and Python build versions have a look at:
+
```
[the remark this answers is not in the document]
Comment (wvangeit) [reply README.md:87]: I've made the change below. @pcrespov, can i resolve this?

[the remark this answers is not in the document]
Comment (pcrespov) [reply README.md:87]: @wvangeit sure! Sorry I did not see this message. Typically when we approve the RP the comments you can resolve them yourself except if you need further interaction.

**Pull request merged**